app/home/home.css

```diff
@@ -1,10 +1,5 @@
 
-.vertical-center {  
-  min-height: 100vh; 
-  display: flex;
-  align-items: center;
-}
-.header{
+/*.header{
```

Comment:
Please, remove the commented code in the files if not used. You can get it later if needed through file history.